source/hardware/controllers/pioneer_ddj_400.rst

```diff
@@ -0,0 +1,185 @@
+- `Hardware Diagram <https://www.pioneerdj.com/-/media/pioneerdj/software-info/controller/ddj-400/ddj-400_hardwarediagram_rekordboxdj_e1.pdf?la=en-us>`__
+- `Mapping Forum Thread <https://mixxx.org/forums/viewtopic.php?f=7&t=12113>`__
+
+.. versionadded:: 2.3.0
+
+Drivers
+-------
+
+You can download the latest firmware from
+`the manufacturer's website <https://www.pioneerdj.com/en/support/software/controller/ddj-400/>`__.
+
+Unfortunately Pioneer offers no Linux support, so to update your controller's
+firmware you will need access to a Windows or Mac machine. Simply download the
+drivers, attach your device and run the installer.
+
+.. note:: It is highlty recommended that you update your controller's firmware.
+          This will fix a bug in the release drivers which causes the tempo
+          sliders to send signals without being touched!
+
+Controller Mapping
+------------------
+
+The schematic drawings used here can be referenced in the
+`original manual <http://docs.pioneerdj.com/Manuals/DDJ_400_DRI1551A_manual/>`__
+on the given page number.
+
+Browser section (p. 6)
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_400_browser.svg
+   :align: center
+   :width: 50%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-400 (browser section)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-400 (browser section)
+
+========  ==================================================  ==========================================
+No.       Control                                             Function
+========  ==================================================  ==========================================
+1         :hwlabel:`LOAD` buttons                             Load song into deck.
+2         Rotary Selector                                     Press to toggle focus between the library sidebar and associated panels. Turn to move focus up or down.
+========  ==================================================  ==========================================
+
+Deck sections (p. 6)
+~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_400_deck.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-400 (deck section)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-400 (deck section)
+
+====  =======================================================  ======================================================================
+No.   Control                                                  Function
+====  =======================================================  ======================================================================
+1     :hwlabel:`BEAT SYNC`                                     Tap to sync tempo to the other playing track. Hold to enable sync lock.
+1     :hwlabel:`SHIFT` + :hwlabel:`BEAT SYNC`                  Cycles through tempo ranges: +/-6%, +/-10%, +/-16%, +/-25%
+2     :hwlabel:`CUE/LOOP CALL`:hwlabel:`>`                     Doubles the currently active loop.
```

Comment:
```suggestion
2     :hwlabel:`CUE/LOOP CALL`:hwlabel:`>`                     Doubles the currently loop size.
```
This doesn't only act when there is a loop active, right?

Reply:
Currently it does, but good point---it doesn't have to stay that way. Adding it to the todo list on the mapping PR.